packages/components/src/DateTimePickers/InputDatePicker/InputDatePicker.component.js

```diff
@@ -85,18 +83,17 @@ export default function InputDatePicker(props) {
 		<DatePicker.Manager
 			value={props.value}
 			dateFormat={props.dateFormat}
-			onChange={(...args) => handlers.onChange(...args, inputRef.current)}
+			onChange={(...args) => handlers.onChange(...args, referenceElement)}
 			useUTC={props.useUTC}
 			timezone={props.timezone}
 		>
 			<FocusManager
 				className={classnames(theme['date-picker'], 'date-picker')}
-				divRef={containerRef}
```

Comment:
This is a ref to set on the div containing the input and the popper. But we don't use it, so let's not set a ref on it here.